server/assets/tests/specs/userApiSpec.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
```

Comment:
Add `process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'` here so it will pass the ignore the ssl